### ISSUE TITLE
Anyway to reconstruct a Sound object from a manipulated pitch? 

### ISSUE THREAD

**Issue by BendingUnit-22** [YannickJadoul/Parselmouth#5]
From the reference page, there's no mention of anyway to replace pitch back to the sound object. In other word,  If I were to set pitch.selected_array['frequency'] to new array of float, can I put the pitch back to the sound object so I can save it as a wav. I notice call([pitch_tier, manipulation], "Replace pitch tier") technically replaces the pitch in the manipulation object, but there's seems to be no way of altering the pitch_tier either than call('Multiply frequencies'). Any help would be appreciated.

**Comment by YannickJadoul** [comment]
@BendingUnit-22 Yeah, Praat makes a different between `Pitch` and `PitchTier` (and `PitchTier` is currently not yet a part of the Python API in Parselmouth), but...
- `PitchTier` does have ways of changing the points in there (cfr. http://parselmouth.readthedocs.io/en/latest/examples/pitch_manipulation.html, and other modifications like "Add point...", "Remove point...", "Modify interval...", "Formula..."; have a look at the Praat docs and menus for more: http://www.fon.hum.uva.nl/praat/manual/PitchTier.html)
- `Pitch` also has the Praat action "Down to PitchTier", which lets you convert a Praat `Pitch` object to `PitchTier`, which lets you do the following:
```Python
pitch = parselmouth.Sound("the_north_wind_and_the_sun.wav").to_pitch()
pitch_tier = parselmouth.praat.call(pitch, "Down to PitchTier")
print(pitch_tier)
```
